ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -152,6 +161,7 @@ descriptor:
   winter: "Method or process used during colder weather (i.e. winter season)."
   zone: "Region of building which is conditioned."
 measurement:
+  fault : "Signal that a fault (low/medium) priority warning is present; can compromise system operation, but not necessarily imminently."
```

Comment:
fault does not belong here, it is not a `measurement`.
can you please move it under `descriptor`?

Reply:
in general, nothing should go under measurements without some type of units asociated with it (e.g. `temperature` --> K, F, etc.).

Additionally: the description for `fault` is similar to what a definition would be for `alarm` (which is currently absent from the ontology). Would there be a distinction between these values in practice, or are all `faults` just `alarms`?

Reply:
fault has been moved to measurement_descriptor.